### PR TITLE
Relabel Project Types to Instance Types in UI

### PR DESCRIPTION
**Pull request by knolleary**
Part of #1689 

This wasn't originally in scope, but having started to write the doc updates for introducing Instances, the Project Type concept stuck out a lot.

This PR is a cosmetic update to the front-end to reference Instance Type instead of Project Type. In some components I've also updated internal variables to be `instance` rather than `project` where the changes are not too extensive. Some views need more detailed going through so I've elected to skip for now.

### CODE REVIEW

**Comment by knolleary** [comment]
The final failing test is due to an update to cypress. @joepavitt has already merged the fix via #1835

**Comment by knolleary** [comment]
Merged main into this branch and all tests are passing. Good for review and merge.

File: frontend/src/pages/admin/InstanceTypes/index.vue [hunk 1]
```diff
@@ -0,0 +1,159 @@
+<template>
```
Comment (Pezmc) [inline frontend/src/pages/admin/InstanceTypes/index.vue:1]: Assuming this file is not new and GitHub hasn't tracked the move

Comment (knolleary) [reply frontend/src/pages/admin/InstanceTypes/index.vue:1]: Correct - I think the combination of rename *and* the number of changes of Project->Instance has exceeded what git thinks is a move and edit.

**Review by Pezmc — APPROVED** [review]
I've read through and checked all the changes, but can't guarantee it's 100%

**Pull request merged**